source/ClientRequest/AMDSClientIntrospectionRequest.cpp

```diff
@@ -94,32 +105,14 @@ bool AMDSClientIntrospectionRequest::readFromDataStream(AMDSDataStream *dataStre
 
 bool AMDSClientIntrospectionRequest::validateResponse()
 {
-	/// TODO: to be added ...
-	return true;
-
-//	QList<AMDSBufferGroupInfo> requestBufferGroupInfos = bufferGroupInfos();
-
-//	if(requestBufferGroupInfos.count() > 0){
-//		const QStandardItemModel* model = qobject_cast<const QStandardItemModel*>(requestType->model());
-//		for(int x = 1; x < 7; x++){
-//			QStandardItem* item = model->item(x);
-//			item->setFlags(Qt::ItemIsSelectable|Qt::ItemIsEnabled);
-//			// visually disable by greying out - works only if combobox has been painted already and palette returns the wanted color
-//			item->setData(QVariant(), Qt::TextColorRole);
-//		}
-//	}
-
-//	if(bufferName() == "All"){
-//		bufferNameComboBox_->clear();
-//		bufferNameComboBox_->addItem("All");
-//	}
-//	for(int y = 0, ySize = requestBufferGroupInfos.count(); y < ySize; y++){
-//		if(bufferName() == "All")
-//			bufferNameComboBox_->addItem(requestBufferGroupInfos.at(y).name());
-//		qDebug() << requestBufferGroupInfos.at(y).name() << requestBufferGroupInfos.at(y).description() << requestBufferGroupInfos.at(y).units() << requestBufferGroupInfos.at(y).rank() << requestBufferGroupInfos.at(y).size().toString();
-//		for(int x = 0, size = requestBufferGroupInfos.at(y).axes().count(); x < size; x++){
-//			qDebug() << "\tAxis info at " << x << requestBufferGroupInfos.at(y).axes().at(x).name() << requestBufferGroupInfos.at(y).axes().at(x).description() << requestBufferGroupInfos.at(y).axes().at(x).units() << requestBufferGroupInfos.at(y).axes().at(x).size() << requestBufferGroupInfos.at(y).axes().at(x).isUniform() << requestBufferGroupInfos.at(y).axes().at(x).start() << requestBufferGroupInfos.at(y).axes().at(x).increment();
-//		}
-//	}
+	for(int y = 0, ySize = bufferGroupInfos_.count(); y < ySize; y++){
```

Comment:
@dretrex as you observed, it doesn't valid anything actually. I didn't change any codes in there. I believe  David added it at the beginning to validate the data.

We might need a better clean up for this later